app.js

```diff
@@ -253,8 +253,13 @@ function createBroadcastFiles(arg) {
 
 const showLine = (line, socket = io) => {
   const overlayPrefs = store.get('obs');
-  const payload = Object.assign(line, overlayPrefs);
-  if (!line.fromScroll) {
+  const lineWithSettings = line;
```

Comment:
Are chromecast changes not part of this PR?

Reply:
Yes, I will send a separate PR for that today.